contracts/BIFI/strategies/Curve/StrategyConvex.sol

```diff
@@ -0,0 +1,375 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin-4/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin-4/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "../../interfaces/common/IUniswapRouterETH.sol";
+import "../../interfaces/common/IWrappedNative.sol";
+import "../../interfaces/curve/ICurveSwap.sol";
+import "../../interfaces/curve/IGaugeFactory.sol";
+import "../../interfaces/curve/IRewardsGauge.sol";
+import "../Common/StratFeeManagerInitializable.sol";
+
+contract StrategyConvex is StratFeeManagerInitializable {
+    using SafeERC20 for IERC20;
+
+    // Tokens used
+    address public want; // curve lpToken
+    address public crv;
+    address public native;
+    address public depositToken;
+
+    // Third party contracts
+    address public gaugeFactory;
+    address public rewardsGauge;
+    address public pool;
+    uint public poolSize;
+    uint public depositIndex;
+    bool public useUnderlying;
```

Comment:
If you move the bools ahead of the uint you actually save a bit of gas since it will pack the bools in the same slot as pool